lib/meal/total.js

```diff
@@ -1,6 +1,12 @@
 var tz = require('moment-timezone');
 var calcMealCOB = require('../determine-basal/cob');
 
+function round(value, digits) {
```

Comment:
Minor comment: It looks like this function is defined in a few different places, I would consider moving it to a shared js file and pulling that in from all of the places that use it.